packages/php/email-editor/composer.json

```
@@ -31,6 +31,9 @@
 		}
 	},
 	"scripts": {
+		"env:destroy": "wp-env destroy",
+		"env:down": "wp-env stop",
```

Just a tiny detail. I would keep the naming aligned and use start stop or alternatively up and down. I think that combining start and down is counterintuitive.
```suggestion
		"env:stop": "wp-env stop",
```